include/binaryfusefilter.h

```diff
@@ -858,4 +860,91 @@ static inline bool binary_fuse8_deserialize(binary_fuse8_t * filter, const char
   return true;
 }
 
+// minimal bitfield implementation
+typedef uint8_t bitf_t;
+#define bitf_w (sizeof(bitf_t) * 8)
```

Comment:
Defines become global, so you either need to prefix them like XOR_bitfw_w or you make sure to undefine them once you are done.

Reply:
Right, I did not consider that we are operating in an included header. Will add the prefix and undefines.